Formula/xh.rb

```diff
@@ -1,9 +1,10 @@
 class Xh < Formula
-  desc "Yet another HTTPie clone"
+  desc "Friendly and fast tool for sending HTTP requests"
   homepage "https://github.com/ducaale/xh"
   url "https://github.com/ducaale/xh/archive/v0.9.1.tar.gz"
   sha256 "b0cd92b428094286688214fde316e0b9bb668c184989a4a0ec25b5ffae2cccd4"
   license "MIT"
+  revision 1
```

Comment:
```suggestion
```
The revision bump isn't needed here. Users don't need this to be re-installed when they do `brew upgrade` from a description update.